src/Examine.Lucene/Suggest/DirectSpellCheckerDefinition.cs

```diff
@@ -0,0 +1,48 @@
+using Lucene.Net.Index;
+using Lucene.Net.Search.Spell;
+using System.Linq;
+using Examine.Suggest;
+
+namespace Examine.Lucene.Suggest
+{
+    public class DirectSpellCheckerDefinition : LuceneSuggesterDefinition, ILookupExecutor
```

Comment:
This is public and should therefore have XML documentation - Use `/// <inheritdoc />` where appropriate

Reply:
Done